README.md

```diff
@@ -100,7 +100,8 @@ the Qt version that comes with the distribution's package management.
       > git clone https://github.com/jahnf/projecteur
       > cd projecteur
       > mkdir build && cd build
-      > QTDIR=/opt/Qt/5.9.6/gcc_64 cmake ..
+      > QTDIR=/opt/Qt/5.9.6/gcc_64 
```

Comment:
Actually that is not a typo. If you want to put it in two seperate lines you would need to do `export QTDIR=/opt/...`, otherwise the next command has no QTDIR variable. Variables that are set inline with a command are set only for that command.

Reply:
You can try this on your shell:
```
MYVAR=abc echo myvar is ${MYVAR}
```

and as two seperate commands
```
MYVAR=abc
echo myvar is ${MYVAR}
```

Reply:
Okay so here is the answer: https://unix.stackexchange.com/questions/126938/why-is-setting-a-variable-before-a-command-legal-in-bash

Reply:
I'd argue that for the sake of readability, it makes some sense to split them...

Reply:
I have to disagree, with your change `cmake` will not have the `QTDIR` variable set.

To make it work you would have to use `export`, like
```
export QTDIR=/path/to/custom/qt
cmake ..
```
And I don't like this, because every other program that gets started after cmake from that shell also has the `QTDIR` set

Reply:
Fair enough. 👍